src/modules/moduleRegistry.tsx

```diff
@@ -1,28 +1,23 @@
-import { PageModulesCollectionFragment } from "@/lib/contentful/fragments/PageModulesCollectionFragment";
 import { ResultOf, TadaDocumentNode } from "gql.tada";
 import { FC } from "react";
 import { ModuleExpertsOverflow } from "./ExpertsOverflow";
 import { GetModuleExpertsOverflowById } from "@/lib/contentful/query/GetModuleExpertsOverflowById";
 import { ModuleCustomerStoriesCarousel } from "./ModuleCustomerStoriesCarousel";
 import { GetModuleCustomerStoriesOverflowById } from "@/lib/contentful/query/GetModuleCustomerStoriesOverflowById";
 
-type ModuleComponent<Data = any> = {
+type ModuleComponent<Data = unknown> = {
   component: FC<{ data: Data }>;
   queryById: TadaDocumentNode<Data, { id: string }>;
 };
 
-type ModuleRegistry = Record<
-  NonNullable<
-    NonNullable<
-      NonNullable<
-        NonNullable<
-          ResultOf<typeof PageModulesCollectionFragment>["items"][number]
-        >["modulesCollection"]
-      >["items"][number]
-    >["__typename"]
-  >,
-  ModuleComponent | null
->;
+type ModuleRegistry = {
```

Comment:
This won't work for what we need. We would actively need to grow this list out, whereas the previous type ensures that we fulfill every type that is within the `PageModulesCollectionFragment`